Improve wording around region matching vs not matching (#496)
powersync-ja/powersync-docs: configuration/source-db/private-endpoints.mdx

2. Provide the following details:
   * **Name**: a recognizable name for the endpoint (for example, `my-private-endpoint`).
   * **Service Name**: the Endpoint Service Name from the previous step (for example, `com.amazonaws.vpce.us-east-1.vpce-svc-0123456`).

>    * **Region**: the PowerSync region to create the Private Endpoint in. This must match the region of the PowerSync instance you plan to use the endpoint with. Note that the region of the underlying AWS PrivateLink Endpoint Service does not need to match.

Reviewer: Maybe lets make it very clear that it's their Endpoint Service we are referring to. ie.
Note that the region of **_your_** underlying AWS PrivateLink Endpoint Service

Author: Good idea thanks

>    * **Region**: the PowerSync region to create the Private Endpoint in. This must match the region of the PowerSync instance you plan to use the endpoint with. Note that the region of the underlying AWS PrivateLink Endpoint Service does not need to match.

Author: ```suggestion
   * **Region**: the PowerSync region to create the Private Endpoint in. This must match the region of the PowerSync instance you plan to use the endpoint with. Note that the region of your underlying AWS PrivateLink Endpoint Service does not need to match.
```

>    * **Region**: the PowerSync region to create the Private Endpoint in. This must match the region of the PowerSync instance you plan to use the endpoint with. The region of the underlying AWS PrivateLink Endpoint Service does not need to match. It can be in any region.

Author: ```suggestion
   * **Region**: the PowerSync region to create the Private Endpoint in. This must match the region of the PowerSync instance you plan to use the endpoint with. Note that the region of the underlying AWS PrivateLink Endpoint Service does not need to match.
```
